apps/freelance/src/pages/PasswordResetRequest/userPasswordResetRequest.ts

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+import {
+  Control,
+  SubmitHandler,
+  useForm,
+  UseFormHandleSubmit,
+} from 'react-hook-form';
+import { SerializedError } from '@reduxjs/toolkit';
+import { usePasswordResetRequestMutation } from 'redux/services/user';
+import { UserError } from 'redux/types/user.types';
+
+type Inputs = {
+  email: string;
+};
+
+interface UsePasswordResetRequestResponse {
+  handleSubmit: UseFormHandleSubmit<Inputs>;
+  onSubmit: (data: Inputs) => SubmitHandler<Inputs>;
+  control: Control<Inputs>;
+  email: string;
+  isLoading: boolean;
+  isError: boolean;
+  error: UserError | SerializedError | undefined;
```

Comment:
`error?: UserError | SerializedError;` should work

Reply:
@lyudad fixed